src/system/Nav/styles.ts

```diff
@@ -75,33 +124,183 @@ export const itemVariantStyle = ( variant: NavVariant ): ThemeUIStyleObject => {
 			};
 		}
 
+		case 'menu': {
+			return {
+				position: 'relative',
+				alignItems: 'center',
+				backgroundColor: 'layer.1',
+				border: 'none',
+				borderRadius: 1,
+				color: 'text',
+				display: 'inline-flex',
+				fontSize: 'inherit',
+				fontWeight: 'body',
+				gap: 3,
+				height: 38,
+				mx: 0,
+				mb: 0,
+				pl: 5,
+				pr: 3,
+				py: 2,
+				textDecoration: 'none',
+				width: '100%',
+				'&:visited': {
+					color: 'text',
+				},
+				'&[data-active]::after': {
+					position: 'absolute',
+					content: "''",
+					overflow: 'hidden',
+					width: 3,
+					backgroundColor: 'borders.accent',
+					borderRadius: '90px',
+					height: 26,
+					top: '6px',
+					left: 3,
+				},
+				'&[data-active]': {
+					color: 'heading',
+					backgroundColor: 'layer.2',
+					textDecoration: 'none',
+					cursor: 'default',
+					svg: {
+						color: 'icon.primary',
+						fill: 'icon.primary',
+					},
+				},
+				'&:focus:not(&[data-active]), &:hover:not(&[data-active])': {
+					color: 'heading',
+					backgroundColor: 'layer.3',
+					svg: {
+						color: 'icon.primary',
+					},
+				},
+				':not(&:hover)': {
+					transition: 'background-color 200ms ease-out',
+				},
+				svg: {
+					color: 'icon.secondary',
+					fill: 'icon.secondary',
+					display: 'block',
+				},
+			};
+		}
+
+		case 'breadcrumbs': {
+			return {
+				...defaultLinkVariantStyle,
+				m: 0,
+			};
+		}
+
+		// Primary and any Other
+		case 'primary':
 		default: {
-			return defaultStyles;
+			return {
+				...baseVariantStyle,
+				variant: `buttons.tertiary`,
+				borderRadius: 1,
+				'&[data-active]': {
+					variant: `buttons.${ variant }`,
+				},
+				'&[aria-disabled="true"]': {
+					opacity: 0.7,
+					color: 'texts.secondary',
+					cursor: 'not-allowed',
+				},
+				':hover': {
+					backgroundColor: `button.${ variant }.background.hover`,
+					textDecoration: 'none',
+				},
+			};
 		}
 	}
 };
 
-export const navItemStyles = ( variant: NavVariant ): ThemeUIStyleObject => {
-	const defaultVariantStyles = itemVariantStyle( variant );
+export const navItemStyles = (
+	orientation: NavProps[ 'orientation' ],
+	variant?: NavVariant
+): ThemeUIStyleObject => {
+	const defaultStyle = {
+		mr: 2,
+		'&:last-of-type': {
+			mr: orientation === 'horizontal' ? 0 : undefined,
+		},
+	};
+
+	switch ( variant ) {
+		case 'menu': {
+			return {
+				...defaultStyle,
+				mr: 0,
+				border: 'none',
+				height: 38,
+				width: '100%',
+				mb: 1,
+			};
+		}
+
+		case 'breadcrumbs': {
+			return {
+				'&::before': {
```

Comment:
@djalmaaraujo minor nitpick but I think it'd be great for people jumping on this code to explain what it does in a short comment. 

This prints the `/` in the breadcrumbs, correct?
![CleanShot 2023-11-21 at 16 19 52@2x](https://github.com/Automattic/vip-design-system/assets/668251/bccf4c3b-ca47-4915-a365-4f0274f23deb)

Reply:
@andrea-sdl Yes, because we should not use the separators as text, according to WCAG suggestions.